src/js/visual/configurator-csr.js

```diff
@@ -1783,22 +1793,24 @@ ripe.ConfiguratorCsr.prototype._registerInitialsHandlers = function() {
  * @ignore
  */
 ripe.ConfiguratorCsr.prototype._unregisterInitialsHandlers = function() {
-    this.owner && this.owner.unbind("initials_extra", this._onInitialsExtraEvent);
-    this.owner && this.owner.unbind("initials", this._onInitialsEvent);
+    this.owner && this.owner.unbind("initials_extra", this._onInitialsExtraEventCallback);
+    this.owner && this.owner.unbind("initials", this._onInitialsEventCallback);
 };
 
 /**
  * @ignore
  */
 ripe.ConfiguratorCsr.prototype._registerConfigHandlers = function() {
-    this.owner.bind("pre_config", (brand, model, options) => this._onPreConfig(this));
-    this.owner.bind("post_config", config => this._onPostConfig(this, config));
+    // unregister previous config handlers to avoid duplicate handlers
+    this._unregisterConfigHandlers();

```

Comment:
Don't call the unregister method here. If you need to do it here, it's a symptom of a bug

Reply:
let's resolve this [here](https://github.com/ripe-tech/ripe-sdk/pull/494#discussion_r1123505246)